app/_hub/kong-inc/prometheus/0.1-x.md

```diff
@@ -186,8 +186,8 @@ allow access to the `/metrics` endpoint to Prometheus.
         location / {
             default_type text/plain;
             content_by_lua_block {
-                local prometheus = require "kong.plugins.prometheus.exporter"
-                prometheus:collect()
+                local serve = require "kong.plugins.prometheus.serve"
+                serve.prometheus_server()
```

Comment:
Let's keep this as `prometheus:collect()` as well. It will work in the older version too.